dojo/filters.py

```diff
@@ -1997,11 +1998,30 @@ class UserFilter(DojoFilter):
     class Meta:
         model = Dojo_User
         fields = ['is_staff', 'is_superuser', 'is_active', 'first_name',
-                  'last_name', 'username']
+                  'last_name', 'username', 'email']
         exclude = ['password', 'last_login', 'groups', 'user_permissions',
                    'date_joined']
 
 
+class ApiUserFilter(DojoFilter):
+    username = ModelMultipleChoiceFilter(field_name='username', to_field_name='username',
```

Comment:
It would be better to have some free form search where a search can be done on any username or part thereof. With the ModelMultipleChoiceFilter you will get a 400 or 500 error if you put in a username that doesn't exist.

Reply:
@valentijnscholten, I'm not sure what kind of search field you expect. How it could cover multiple usernames in one request.

Reply:
I believe he is asking for just a text field rather than a dropdown to select the user. These filters work with http query so an invalid query would result in a 4/500.

Heres an example `username = CharFilter(field_name='username', lookup_expr='icontains')` so that if I query for a user 'kib', then your username will be returned. If there are multiple users with 'kib' in their username, then they will also appear in the filtered results.

Reply:
I think you could achieve this by copying the CharFieldInFilter I implemented for multiple tag search:
```
    tags = CharFieldInFilter(field_name='tags__name', lookup_expr='in',
                             help_text='Comma seperated list of exact tags')
```